javascripts/discourse/initializers/init-topic-excerpts.js

```diff
@@ -1,8 +1,18 @@
 import { apiInitializer } from "discourse/lib/api";
+import { withSilencedDeprecations } from "discourse-common/lib/deprecated";
 
 export default apiInitializer("0.8", (api) => {
-  api.modifyClass("component:topic-list-item", {
-    pluginId: "discourse-air",
-    expandPinned: true,
-  });
+  const transformerExists = api.registerValueTransformer(
+    "topic-list-item-expand-pinned",
+    () => true
+  );
+
+  if (!transformerExists) {
```

Comment:
Transformer was only introduced a couple of days ago, and there hasn't been a version bump since, so the d-compat entry won't work for every case. Adding some conditional logic which falls-back to the modifyClass instead.

withSilencedDeprecations is mostly included for grep-ability when we come to clean up themes/plugins after the topic-list conversion is complete.